tests/integration/awslambda/functions/dotnet6/src/dotnet6/Readme.md

```diff
@@ -0,0 +1,49 @@
+# AWS Lambda Empty Function Project
```

Comment:
It would be nice to rework this readme (remove all the autogenerated stuff), with just a short notice on how to build the commited zip archive (should be something like `dotnet lambda package`, right?)

Reply:
yup, I updated the readme.